lucid-book-store/src/components/navbar/navbar.jsx

```diff
@@ -25,7 +25,31 @@ const NavbarCartButton = ({ cartNotification }) => (
   </button>
 );
 
+const NavbarLoginButton = ({ token, toggleLogoutModal }) => {
+  const navigate = useNavigate();
+  if (token) {
+    return (
```

Comment:
instead of having two conditional returns, it would be better if you return only once and render the buttons according to the availability of the tokens.

Reply:
Thanks for pointing that out will, do that